aws_lambda_functions/CloudResourceOrchestration/lambda_function.py

```diff
@@ -0,0 +1,101 @@
+import json
```

Comment:
Why did you call it lambda function ?

Reply:
It is written in the lambda function document. I followed it.